### PR TITLE
Translate comments in SpellAuras.cpp from French to English

### PR DESCRIPTION
**Pull request by DeltaF1**
## 🍰 Pullrequest
Working on translating aura descriptions from french to english

### Issues
- relates [#683](https://github.com/vmangos/core/issues/683)

### Todo / Checklist
- [x] Random code comments
- [x] _IsExclusiveSpellAura buff list comments

### CODE REVIEW

**Comment by 0blu** [comment]
Just in case you want to translate even more stuff:
In `*commands.cpp` there are a lot of  `"oui"/"non"` instead of  `"yes" /"no"` .

(_Reminds me of the Futurama gag <https://www.youtube.com/watch?v=pwODwwgE6rA>_ 🤣)

**Comment by DeltaF1** [comment]
> In *commands.cpp there are a lot of "oui"/"non" instead of "yes" /"no" .

What does PSendSysMessage do? Are there scripts or something that depend on those values being OUI/NON?

**Comment by 0blu** [comment]
`PSendSysMessage` is just a function that sends a message back to the player.
There should be no conflict with any script or other dependency.

**Comment by DeltaF1** [comment]
I know that people do things like write scripts to parse combat log messages which is why I asked. I'll make a separate PR for it just in case.

**Comment by DeltaF1** [comment]
@0blu Are you able to review this PR? No rush at all, I just want to make sure it's not in limbo since it started as a draft

**Comment by ratkosrb** [comment]
I don't think he speaks French to be able to review it.

File: src/game/Spells/SpellAuras.cpp [hunk 1]
```diff
@@ -952,6 +952,7 @@ bool Aura::CanProcFrom(SpellEntry const* spell, uint32 EventProcEx, uint32 procE
     uint64 mask = sSpellMgr.GetSpellAffectMask(GetId(), GetEffIndex());
 
     // Nostalrius: c'est la moindre des choses d'utiliser un peu 'spell_proc_event' non ?
```
Comment (0blu) [inline src/game/Spells/SpellAuras.cpp:954]: Can you please replace this too, no need to put the translation below it.
Although I am unsure what this means in this context.
@ratkosrb maybe you can give us a bit of context.

File: src/game/Spells/SpellAuras.cpp [hunk 2]
```diff
@@ -4268,7 +4271,7 @@ void Aura::HandleModMechanicImmunityMask(bool apply, bool /*Real*/)
     // check implemented in Unit::IsImmuneToSpell and Unit::IsImmuneToSpellEffect
 }
 
-//this method is called whenever we add / remove aura which gives m_target some imunity to some spell effect
+//this method is called whenever we add / remove aura which gives m_target some immunity to some spell effect
```
Comment (0blu) [inline src/game/Spells/SpellAuras.cpp]: Adding a space
```suggestion
// this method is called whenever we add / remove aura which gives m_target some immunity to some spell effect
```

**Review by 0blu — COMMENTED** [review]
Unfortunately I don't speak French, but I used deepL to get a second opinion on the translations.
Other than my addition your changes look good.
Many thanks for your effort!

File: src/game/Spells/SpellAuras.cpp (hunk 1 above)
Comment (DeltaF1) [reply src/game/Spells/SpellAuras.cpp:954]: Yeah I was confused about what this means. I took it as more of a conversational note rather than explaining what the code was doing, so I wasn't sure whether to translate it at all or just leave it as-is

File: src/game/Spells/SpellAuras.cpp [hunk 3]
```diff
@@ -8430,70 +8433,69 @@ bool _IsExclusiveSpellAura(SpellEntry const* spellproto, SpellEffectIndex eff, A
...
-        case 25661: // Bouffe +25 Endu
-        case 24427: // Diamond Flask
-        case 17528: // Mighty Rage Potion
+        case 22730: // Increased Intellect
+        case 18141: // Blessed Sunfruit Juice
+        case 18125: // Blessed Sunfruit
+        case 18192: // Increased Agility
+        case 18191: // Increased Stamina
+        case 25661: // Increased Stamina
+        case 24427: // Toasty
```
Comment (Daribon) [inline src/game/Spells/SpellAuras.cpp:8486]: Why was this changed to Toasty? Spell 24427 is Diamond Flask.
There is no spell named Toasty in vanilla.

Comment (DeltaF1) [reply src/game/Spells/SpellAuras.cpp:8486]: I pulled the data for these comments straight from the vmangos database. It looks like it was called Toasty in one of the older builds. I can try to write a query to only pull the latest names if that's most useful.

| name | build |
|--------|--------|
| Toasty | 4695 |
| Diamond Flask | 4878 | 
| Diamond Flask | 5302 |
